lib/whitelistedUrlPrefixes.js

```diff
@@ -23,9 +23,7 @@ module.exports = [
   'https://balance-staging.mercury.basicattentiontoken.org/',
   'https://publishers.basicattentiontoken.org/',
   'https://publishers-staging.basicattentiontoken.org/',
-  'https://updates.bravesoftware.com/', // remove this once updates are moved to the prod environment
-  'https://ads-serve.bravesoftware.com/', // remove this once ads catalog moves to using prod
```

Comment:
what's the prod url for this? just curious. @amirsaber

Reply:
`https://ads-serve.brave.com/v3/catalog` is the prod URL according to https://bravesoftware.slack.com/archives/CJR5902AW/p1579288365002100

Reply:
should that be added to this list then?

Reply:
nvm, just saw your comment about how the tests pass regardless. i wonder why it's not failing then? are ads not downloaded upon first browser startup?

Reply:
maybe the issue is that rewards needs to be enabled before the catalog is downloaded, and this test doesn't automatically go through the onboarding process

Reply:
> maybe the issue is that rewards needs to be enabled before the catalog is downloaded, and this test doesn't automatically go through the onboarding process

That's my guess as well.